Makefile

```diff
@@ -269,13 +269,13 @@ test-cov: all
 	$(MAKE) lint
 
 test-parallel: all
-	$(PYTHON) tools/test.py $(PARALLEL_ARGS) --mode=release parallel
+	$(PYTHON) tools/test.py $(PARALLEL_ARGS) --mode=$(BUILDTYPE_LOWER) parallel
 
 test-valgrind: all
-	$(PYTHON) tools/test.py $(PARALLEL_ARGS) --mode=release --valgrind sequential parallel message
+	$(PYTHON) tools/test.py $(PARALLEL_ARGS) --mode=$(BUILDTYPE_LOWER) --valgrind sequential parallel message
 
 test-check-deopts: all
-	$(PYTHON) tools/test.py $(PARALLEL_ARGS) --mode=release --check-deopts parallel sequential
+	$(PYTHON) tools/test.py $(PARALLEL_ARGS) --mode=$(BUILDTYPE_LOWER) --check-deopts parallel sequential
```

Comment:
This change is also needed in `vcbuild.bat`. I think `node-check-deopts` might be the only one in `vcbuild.bat` that has `--mode=release` hard-coded. So just one line needs to change in `vcbuild.bat` I think.

Reply:
@Trott I don't think there is a equivalent of `BUILDTYPE_LOWER` in `vcbuild.bat`, the `test_args` are manipulated in a different way. I suspect `node-check-deopts` is running the tests twice and the first round can be replaced by adding `--check-deopts` to `test_args`, but that probably belongs to another PR.